### DISCUSSION TITLE
Some questions for beginners to openfast

### DISCUSSION THREAD

**Discussion by zhangtopus** [OpenFAST/openfast#1758] [Q&A]
Hello,I am a beginner ,now I have learnt how to run OpenFast and get some results ,but I have trouble in learning it in advance.Under my given wind speed conditions, the results obtained are similar to the restart time when it is close to 10000s.Could you please give me some advice or guide ,thank you very much.
![微信图片_20230907001020](https://github.com/OpenFAST/openfast/assets/97727163/df316dbc-5686-48d2-bdb6-b29fc476886e)

![微信图片_20230907000900](https://github.com/OpenFAST/openfast/assets/97727163/f61dac7e-bc34-4245-a36b-891bc9eb54cb)

**Comment by jjonkman** [comment]
Dear @zhangtopus,

I'm not understanding your question.  Please clarify.

Best regards,

**Reply by zhangtopus** [reply]
Dear @jjonkman ,
When I input a completely symmetric time series, the bts file is obtained through TurbSim; use this file as the input condition. The results obtained showed that the dynamic response suddenly changed at nearly 10,000 seconds and continued to be 0. Is it because the data after 10000s cannot be read, or is it due to other reasons?Could you please give me some advice or guide ,thank you very much.

**Reply by jjonkman** [reply]
Dear @zhangtopus,

I would guess that you have something in your OpenFAST model that is being triggered at around 10000 seconds, for example, a controller override maneuver in ServoDyn that we often set to 9999.9 s in the input file, which is larger than most cases we simulate (so as to not be triggered).

Best regards,

**Reply by zhangtopus** [reply]
My problem was solved very well. Thank you very much.
